tests/foreman/api/test_syncplan.py

```diff
@@ -607,8 +607,12 @@ def validate_repo_content(
             except AssertionError:
                 sleep(30)
         else:
-            raise AssertionError(
-                'Repository contains invalid number of content entities')
+            repo = repo.read()
+            self.assertNotEqual(
```

Comment:
Correct me if I am wrong, but shouldn't it be `self.assertEqual` instead?
1) We tried to fetch repository content for `max_attempt` times
2)  If none content is present we going to `else`
3) if it is still None then raise exception